### PR TITLE
Add mosfit

### PR DESCRIPTION
**Pull request by guillochon**

### CODE REVIEW

File: recipes/mosfit/meta.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{% set name = "mosfit" %}
+{% set version = "0.4.7" %}
+{% set sha256 = "e63f1196524f0d59915fc8289cd8bd023999853e59ccf54254fc8783112ab901" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ mosfit }}-{{ 0.4.7 }}.tar.gz
```
Comment (pkgw) [inline recipes/mosfit/meta.yml]: Think-o here: should be `{{ name }}-{{ version }}.tar.gz`.

File: recipes/mosfit/meta.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - inflect
+    - matplotlib
+    - mpi4py
+    - nbstripout
+    - numexpr
+    - palettable
+    - seaborn
+    - six
+  run:
+    - python
```
Comment (pkgw) [inline recipes/mosfit/meta.yml]: All of your build-time dependencies should be run-time dependencies as well. The first group has things that you need installed to produce the package; the second has things that users need to have installed in order to for the package to be functional.

File: recipes/mosfit/meta.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+test:
+  imports:
+    - mosfit
+
+about:
+  home: http://github.com/guillochon/mosfit
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
```
Comment (pkgw) [inline recipes/mosfit/meta.yml]: Needs updating.

File: recipes/mosfit/meta.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+  description: |
+    mosfit is a package that performs fitting of semi-analytical model predictions
+    to observed transient data. Data can be provided by the user, or can be pulled
```
Comment (pkgw) [inline recipes/mosfit/meta.yml]: "transient data" is confusing if you don't know the astrophysical context; I suggest clarifying this.

**Comment by guillochon** [comment]
@pkgw Any idea why the linter failed to find the recipe?

**Comment by pkgw** [comment]
Ohhhhh. `meta.yaml`, not `meta.yml`. I bet that's it.

**Comment by pkgw** [comment]
That was fast!

**Comment by guillochon** [comment]
@pkgw so the CI claims that Cython is missing in their channel, but I appear to have it in the conda-forge channel. Do I need to specify the channel in build/run?

**Comment by pkgw** [comment]
No, that's all taken care of. I think it's a capitalization issue: the Conda package name is lower-case `cython`. (When in doubt, do a `conda search <name>` to see what's available.

**Comment by guillochon** [comment]
Strangely it works with the camel case on my machine...

**Comment by pkgw** [comment]
OK, next problem is your `license_file` specification. It should point to a file in your source distribution; looks like it should just be `LICENSE`, not `LICENSE.txt`.

**Comment by pkgw** [comment]
How are you generating the tarballs that you upload to PyPI? I see some `.DS_Store` files in the archives, which shouldn't be there. I also suspect that the `docs/_build` directory would ideally not be included either.

**Comment by guillochon** [comment]
`python setup.py sdist upload`. The `.DS_Store` files are created on OS X in all directories. `_build` and `.DS_Store` are both in my .gitignore, not sure how to avoid getting them sucked up in the setup...

**Comment by guillochon** [comment]
Ah looks like I have to exclude them in `MANIFEST.in`.

**Comment by pkgw** [comment]
Aaaand conversely, LICENSE *isn't* getting distributed in the source tarballs, unless you added it in the 0.4.8 release you just made ...

**Pull request closed without merging**